src/applications/gqrx/file_resources.cpp

```diff
@@ -32,7 +32,7 @@ std::string receiver::get_random_file(void)
     static std::string path;
     if (path.empty())
     {
-        path = "/dev/urandom";
+        path = "/dev/zero";
```

Comment:
The name of this method should be changed to reflect its new purpose. Also, line 45 should be changed to fill the file with zeroes.

Reply:
Thanks.
Reverted this change in a8e9e898695b64d852a9ef4a4537ffd7c6473bc4.

Reply:
This has been done in #1001